src/main/java/com/aws/greengrass/mqttbridge/MQTTBridge.java

```diff
@@ -91,27 +94,28 @@ protected MQTTBridge(Topics topics, TopicMapping topicMapping, MessageBridge mes
 
     @Override
     public void install() {
-        mappingConfigTopics =
-                this.config.lookupTopics(KernelConfigResolver.CONFIGURATION_CONFIG_KEY, MQTT_TOPIC_MAPPING);
-
-        mappingConfigTopics.subscribe((whatHappened, node) -> {
-            if (mappingConfigTopics.isEmpty()) {
-                logger.debug("Mapping empty");
-                topicMapping.updateMapping(Collections.emptyMap());
-                return;
-            }
+        this.config.lookupTopics(CONFIGURATION_CONFIG_KEY).subscribe((whatHappened, node) -> {
+            if (!skipUpdatingMqttTopicMapping(whatHappened, node)) {
```

Comment:
Nit: Can we un-invert this? Easier to read IMO

Reply:
Fixed